openerp/addons/base/ir/ir_cron.py

```diff
@@ -87,7 +87,7 @@ def method_direct_trigger(self, cr, uid, ids, context=None):
             context={}
         cron_obj = self.browse(cr, uid, ids, context=context)
         for cron in cron_obj:
-            self._callback(cr, uid, cron_obj.model, cron_obj.function, cron_obj.args, cron_obj.id)
+            self._callback(cr, SUPERUSER_ID, cron_obj.model, cron_obj.function, cron_obj.args, cron_obj.id)
```

Comment:
I share the same POV as @guewen. Moreover using the SUPERUSER_ID breaks the multi company rules based on record rules. For sure this code will introduce bugs!